### PR TITLE
adding partial thread safety to encoding_error

### PR DESCRIPTION
**Pull request by b-maldoca**

### CODE REVIEW

File: maldoca/base/encoding_error.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC
+// Copyright 2020 Google LLC
```
Comment (dtao-oss) [inline maldoca/base/encoding_error.h:1]: 2021?

File: maldoca/base/encoding_error.cc [hunk 2]
```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google LLC
```
Comment (dtao-oss) [inline maldoca/base/encoding_error.cc:1]: 2021

**Review by dtao-oss — APPROVED** [review]

File: maldoca/base/encoding_error.cc (hunk 2 above)
Comment (b-maldoca) [reply maldoca/base/encoding_error.cc:1]: done

File: maldoca/base/encoding_error.h (hunk 1 above)
Comment (b-maldoca) [reply maldoca/base/encoding_error.h:1]: done

**Pull request merged**